src/entities/export.ts

```diff
@@ -5,10 +5,71 @@ import db from "../db";
 import * as schema from "../schema";
 import { serializeAccount } from "./account";
 import { serializeList } from "./list";
-import { getPostRelations, serializePost } from "./status";
+import { getPostRelations } from "./status";
 
+const homeUrl = process.env["HOME_URL"] || "http://localhost:3000";
+
+export const serializePost = (post: Post, actor: {id: ActorIdType}) => {
+  const note = {
+    "@context": "https://www.w3.org/ns/activitystreams",
+    id: new URL(post.iri),
+    type: "Note",
+    summary: post.summary || null,
```

Comment:
I'm fairly sure `null` is not allowed in most fields here. This should probably be `post.summary || undefined` or something similar.

Reply:
I think `null` is allowed per https://www.w3.org/TR/activitystreams-core/#syntaxconventions

Omitting the entry and setting to `null` have slightly different semantics.

Either is probably kind of fine here.